package.json

```
@@ -26,7 +26,7 @@
     "configcat-js": "^8.0.0",
     "copy-anything": "^3.0.5",
     "imurmurhash": "^0.1.4",
-    "json-logic-js": "^2.0.2",
+    "json-logic-engine": "^1.3.1",
```

```suggestion
    "json-logic-engine": "1.3.1",
```

This is a bit of an unpopular dependency. We don't HAVE TO but we could lock the version here.